contracts/gov/src/contract.rs

```diff
@@ -627,6 +641,14 @@ pub fn cast_vote(
         return Err(ContractError::InsufficientStaked {});
     }
 
+    let veanc_voting_power = query_user_voting_power(
+        deps.as_ref(),
+        config.anchor_voting_escrow,
+        &sender_address_raw,
+    )?;
+
+    let amount = amount + veanc_voting_power;
```

Comment:
how are we determining `quorum` here? it seems that the `veanc_voting_power` will be counted in the vote counts, but not counted in the total amount that could have voted, which will result in an incorrect calculation of `quorum`